src/main/java/org/prebid/server/cookie/UidsCookieService.java

```diff
@@ -124,6 +139,15 @@ public Cookie toCookie(UidsCookie uidsCookie) {
         return cookie;
     }
 
+    /**
+     * Returns the Uid with the closest expiration date, e.i. the one that will expire sooner.
+     */
+    private static Map.Entry<String, UidWithExpiry> getClosestExpiration(Map.Entry<String, UidWithExpiry> first,
+                                                                         Map.Entry<String, UidWithExpiry> second) {
+        return first.getValue().getExpires().toEpochSecond() < second.getValue().getExpires().toEpochSecond()
```

Comment:
Instead of using `toEpochSecond` consider using `isBefore`

Reply:
Good call, thanks!